po/glossary/fr.po

```diff
@@ -2,18 +2,21 @@
 # Copyright (C) 2006,2013 Free Software Foundation, Inc.
 # Jonathan Ernst <jonathan@ernstfamily.ch>, 2006
 # Sébastien Villemot <sebastien@debian.org>, 2013
+# La Boussole <yoann@laboussole.coop>, 2018.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: gnucash 2.0\n"
 "POT-Creation-Date: 2017-10-20 19:04+0200\n"
-"PO-Revision-Date: 2013-12-25 13:16+0100\n"
-"Last-Translator: Sébastien Villemot <sebastien@debian.org>\n"
-"Language-Team: FR\n"
+"PO-Revision-Date: 2018-10-25 12:30+0200\n"
+"Last-Translator: La Boussole <yoann@laboussole.coop>\n"
+"Language-Team: français <yoann@laboussole.coop>\n"
 "Language: French\n"
```

Comment:
It is common to use the language code here, but I believe, for the glossaries we have it always wrong. If I do not forget it, I will review and update all.

https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html

Reply:
If you're referring to the "Language-Team line, that's for the Translation Project. Independent translators like @YBSLE should probably just delete that line. Does msgfmt complain if it's not there?

Reply:
No, I was talking about "Language:...".

For de - also no TP team - we use
"Language-Team: GnuCash-de <gnucash-de@gnucash.org>\n"
and I added it recently to translator-credits, so users can see it, too.